src/circular-progress.jsx

```diff
@@ -72,15 +72,17 @@ const CircularProgress = React.createClass({
     this._rotateWrapper(wrapper);
   },
 
+  componentWillUnmount() {
+    clearTimeout(this._scalePathTimer);
```

Comment:
My point is that with react, the only thing that is supposed to be public is the property. So, that `_` don't really makes sense to me.